app/views/earnings_and_benefits/edit.html.slim

```diff
@@ -0,0 +1,108 @@
+/ TODO: Add required fields
+/ TODO: Check if there are "hidden" elements on the Jadu form
+.content-body.earnings-and-benefits
+  - content_for :form_page_title do
+    = "Earnings and Benefits"
+
+  = form_for @earnings_and_benefits, url: earnings_and_benefits_path, method: :patch  do |f|
+    #question1.form-group
+      fieldset
+        legend
+          h1.heading-medium Are the claimant's hours of work correct?
+        / Radio button for 'Yes'
+        .multiple-choice
+          = f.radio_button(:agree_with_claimants_hours, true, class: 'form-control')
+          = f.label(:agree_with_claimants_hours, 'Yes', class: 'form-label')
+        / Radio button for 'No'
+        .multiple-choice data-target="disagree_claimants_hours"
+          = f.radio_button(:agree_with_claimants_hours, false, class: 'form-control')
+          = f.label(:agree_with_claimants_hours, 'No', class: 'form-label')
+        / Hidden field for disagreement reason, shows when radio button is clicked
+        #disagree_claimants_hours.panel.panel-border-narrow.js-hidden
+          = f.label(:queried_hours, 'Please enter the details you believe to be correct', class: 'form-label')
+          = f.text_field(:queried_hours, class: 'form-control')
+          span hours each week
+    #question2.form-group
+      fieldset
+        legend
+          h1.heading-medium Are the earnings details given by the claimant correct?
+        / Radio button for 'Yes'
+        .multiple-choice
+          = f.radio_button(:agree_with_earnings_details, true, class: 'form-control')
+          = f.label(:agree_with_earnings_details, 'Yes', class: 'form-label')
```

Comment:
Should this not be agree_with_earnings_details_true ?

Reply:
I was going to leave this until my next run on this page but you're right, should be done now.

Please see commit 6464ad7, if I've missed any I can collect and amend later.